repo-template-terraform.tf

```diff
@@ -0,0 +1,35 @@
+resource "github_repository" "dataworks-terraform-repo-template" {
+  name        = "dataworks-terraform-repo-template"
+  description = "Template Terraform repository for DataWorks GitHub"
+  auto_init   = true
+  is_template = true
+
+  allow_merge_commit = false
+  has_issues         = true
```

Comment:
Point of contention this, I think. I don't like us being able to manage issues in 2 places, so I think this should be false. I know @dan-hill2802 has a dissenting opinion though. I do like the idea of managing issues with the code, but that's not where we're at now, or likely to ever get to. JIRA, for better or worse, is how we manage issues.

Reply:
Issues aren't just for us, we should allow other consumers to report issues